lib/NetPacks.h

```diff
@@ -1087,8 +1087,8 @@ struct DLL_LINKAGE BulkMoveArtifacts : CArtifactOperationPack
 
 		LinkedSlots() = default;
 		LinkedSlots(const ArtifactPosition & srcPos, const ArtifactPosition & dstPos)
-			: srcPos(srcPos)
-			, dstPos(dstPos)
+			: srcPos(std::move(srcPos))
+			, dstPos(std::move(dstPos))
```

Comment:
I might be wrong, but I think std::move here is pointless since parameters are passed as const reference.
You should use either:
`Foo(const Bar & bar) : bar(bar)`
`Foo(Bar bar) : bar(std::move(bar))`

Reply:
Yes you are right. Removed